secureli/services/language_support.py

```diff
@@ -189,21 +192,25 @@ def create_repo(raw_repo: dict) -> Repo:
         repos = [create_repo(raw_repo) for raw_repo in config.get("repos", [])]
         return HookConfiguration(repos=repos)
 
-    def _build_pre_commit_config(self, languages: list[str]) -> BuildConfigResult:
+    def _build_pre_commit_config(
+        self, languages: list[str], lint_languages: set[str]
```

Comment:
Why is `languages` a list, but `lint_languages` is a set? Should we make both of them sets? (Or are you just considering that out of scope for this ticket?)

Reply:
It's probably unnecessary but I made `lint_languages` a set for when it loops through `languages`  [here](https://github.com/slalombuild/secureli/pull/332/files#diff-0de154eba7235013d18401bb2505720dc0c465cc3a70295527f4cbf1ff32af7fR212) and checks if the `language` is in the `lint_languages`. It could probably just be an array since both will be short in length.

Reply:
I assumed you wanted a set for deduplication purposes. I think either one is fine, but I would encourage consistency either way.

Reply:
Good call, updated it to a list